src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+const getPartnerText = (person) => {
+  const maleSex = 'm';
```

Comment:
Static constants should be placed outside the component definition and called uppercase:
```suggestion
  const MALE = 'm';
```